internal/serverinstall/nomad.go

```diff
@@ -201,14 +211,14 @@ EVAL:
 	}, nil
 }
 
-// waypointNomadJob takes in a nomadConfig and returns a Nomad Job per the
+// newWaypointNomadJob takes in a nomadConfig and returns a Nomad Job per the
 // Nomad API
-func waypointNomadJob(c nomadConfig) *api.Job {
-	job := api.NewServiceJob("waypoint-server", "waypoint-server", c.region, 50)
+func newWaypointNomadJob(c nomadConfig) *api.Job {
```

Comment:
I think this name should go back only because I'm using this function in Upgrades to submit the same job config to the current running Nomad job, so it might be confusing going forward since it's also the same config for updating a job.

Or maybe it could be renamed `waypointNomadJobConfig`?

Reply:
Oh good call. I'll just move it back to avoid those fun merge conflicts we're bound to run in to 😄